All features

Danie12345 opened the pull request:
Feature includes bug fixes and features:
- Fixed issue where adding new items would uncheck already checked items (visually). Problems in re-rendering.
- Fixed issue where trying to remove items would make the delete button disappear.

- Add is fully functional.
- Remove is fully functional.
- Clear selected is fully functional.
- Automatic removal when a task is empty is fully functional.

Danie12345 committed:
Add input updating

Danie12345 committed:
Remove all selected tasks

Danie12345 committed:
Fixed list minimum height

Danie12345 committed:
Fix remove individual item

Danie12345 committed:
Fix unchecked items when adding new item

Danie12345 committed:
Fix linter errors

Danie12345 committed:
Update /dist build

Danie12345 committed:
Merge pull request #9 from Danie12345/add-remove-items

Add remove items

FrederickMih commented on src/modules/list.js:
```diff
@@ -39,6 +44,14 @@ export default class List {
+  removeSelected() {
+    Object.values(this.list).forEach((item) => {
+      if (item.completed) {
+        this.removeItem(item);
```
- [x] This actually works but let's stick to the requirements.

You are supposed to implement a separate function for the `Clear all Completed` button using `Array.filter()` 
method as the requirement says. Please let's fix this issue and move a step toward approval instead of still 
using the same `removeItem` method previously defined above to remove all completed items.

Danie12345 committed:
Fix small bug on adding item

Update /dist build

FrederickMih reviewed (changes requested):
Hi @Danie12345 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

# To Highlights

-  Linters checks are all passing with green check marks :heavy_check_mark: 
- Updated array of items is stored in the local storage as expected :+1: 

## Required Changes ♻️

- I'm leaving this comment here because you didn't make any changes to the README file.

- [x] Please let's make sure that our README file is descriptive and professional. The description of your project is completely different from the app you are building. This is not a booklist project.
- [x] The Author section of your README doesn't contain links to your LinkedIn and Twitter accounts which are important for recruiters to have access to your supplementary information.
- [x] The MIT licensed link is broken. 

Let's kindly fix these issues, please.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Danie12345 committed:
Fix bugs

Fix rendering issues implemented in previous fix.

Fix build.

Change removing items using filter.

Fix README.md links and description.

AmaduKamara reviewed (approved):
# Status: Approved 🟢🟢🟢 

Hi @Danie12345,

Good job fixing all the changes asked by the previous reviewer. Your project is working fine 💪💪💪

![MissionCompleteGIF (2)](https://user-images.githubusercontent.com/50941074/165864352-c9d2eefc-f4ca-4aab-9d22-1a07128603ff.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
